test/verify/check-system-realms

```diff
@@ -1211,7 +1211,7 @@ class TestPackageInstall(packagelib.PackageCase):
         self.login_and_go("/system")
 
         # Joining a domain should bring up the install dialog
-        b.wait_text(self.domain_sel, "Join domain")
+        b.wait_text(self.domain_sel, "Install realmd support")
```

Comment:
I am not quite sure I understand, by label are you referring to the "Join Doman" text in the tests.
Because (after the changes we discussed) only one test needs to really be Updated for now

Reply:
Gotcha, I've made the change in case 3 as well.